docs/source/guides/performance.rst

```diff
@@ -35,9 +61,54 @@ We can also set chunk size to be a percentage of total rows::
                                            target_entity="customers",
                                            chunk_size=.05)
 
+Using persistent cluster
+************************
+Behind the scenes, Featuretools uses `Dask's <http://dask.pydata.org/>`_ distributed scheduler to implement multiprocessing. When you only specify the ``n_jobs`` parameter, a cluster will be created for that specific feature matrix calculation and destroyed once calculations have finished. A drawback of this is that each time a feature matrix is calculated, the entity set has to be transmitted to the workers again. To avoid this, we would like to reuse the same cluster between calls. The way to do this is by creating a cluster first and telling featuretools to use it with the ``dask_kwargs`` parameter::
+
+    import featuretools as ft
+    from dask.distributed import LocalCluster
+
+    cluster = LocalCluster()
+    fm_1 = ft.calculate_feature_matrix(features=features_1,
+                                       entityset=entityset,
+                                       cutoff_time=cutoff_time,
+                                       dask_kwargs={'cluster': cluster},
+                                       verbose=True)
+
+The 'cluster' value can either be the actual cluster object or a string of the address the cluster's scheduler can be reached at. The call below would also work. This second feature matrix calculation will not need to resend the entityset data to the workers because it has already been saved on the cluster.::
+
+    fm_2 = ft.calculate_feature_matrix(features=features_2,
+                                       entityset=entityset,
+                                       cutoff_time=cutoff_time,
+                                       dask_kwargs={'cluster': cluster.scheduler.address},
+                                       verbose=True)
+
+.. note::
+
+    When using a persistent cluster, Featuretools publishes a copy of the ``EntitySet`` to the cluster the first time it calculates a feature matrix. Based on the ``EntitySet``'s metadata the cluster will reuse it for successive computations. This means if two ``EntitySets`` have the same metadata but different row values (e.g. new data is added to the ``EntitySet``), Featuretools won’t recopy the second ``EntitySet`` in later calls. A simple way to avoid this scenario is to use a unique ``EntitySet`` id.
+
+Using the distributed dashboard
```

Comment:
i think this header can go one level down since it is related to "Simple Parallel Feature Computation"